### DISCUSSION TITLE
Enterprise Roundup September 2024

### DISCUSSION THREAD

**Discussion by queenofcorgis** [community/community#139749] [Enterprise]
![Header of Enterprise category features the text Enterprise Roundup on the upper lefthand corner on a black background with an abstract peach and teal abstract line design](https://github.com/user-attachments/assets/7f286656-ba58-49ef-857a-4bd9ea7e4c4c)

School definitely felt back in session in the Enterprise category this month, from curious questions being asked by our members to all the learning being done in the GitHub Advanced Security certification course - learning was in the air. Let’s keep learning about what’s new for GitHub Enterprise and your GitHub journey in this month’s check-in. 

### 🔍 Curious Questions:  Quality questions asked in the past month! 
* [Can Azure Bulk Credits Be Applied to GitHub Copilot Billing via Azure Subscription?](https://github.com/orgs/community/discussions/137659) - find out the answer to this great question from @khakie
* Are you looking to upgrade your GitHub Enterprise server? Check out the resource shared inspired by [this discussion](https://github.com/orgs/community/discussions/137901) from @waqar01? 
* [How to restrict and validate member invitations?](https://github.com/orgs/community/discussions/138019) @pubbles1 asked an interesting question about preventing organization owners from accidentally inviting the wrong person. Have a solution or done something similar? Join [the discussion](https://github.com/orgs/community/discussions/138019) to share the knowledge. 

### 📣 Announcements: The latest updates around the Community and GitHub 🆕
* **Git your learning on with GitHub Advanced Security Certification**: We’ll be offering additional practice, resources, and expertise throughout the month of September to get you prepared for the GitHub Advanced Security certification. 

   **Five folks who participate in the prep will be selected to receive a free certification exam voucher.** All four weeks of prep materials and questions are now live in[ this Discussion](https://github.com/orgs/community/discussions/137197)!
* **Join Us at GitHub Universe 2024 - Unleashing the Future of Software Development**: You're invited to join GitHub Universe—our flagship product and community event—October 29-30 live at the historic Fort Mason Center in San Francisco.  Whether you're a seasoned developer, a tech entrepreneur, or an IT leader, GitHub Universe 2024 offers something for everyone. Don't miss out on this unique opportunity to be part of the future of software development. [Secure your spot today](https://githubuniverse.com/?utm_source=event&utm_medium=rm&utm_campaign=sales) and join us for an unforgettable experience!

### 🗳️ Recent Releases: All updates, info, releases, etc that are Enterprise Server and Cloud -related!
* **Enterprise audit logs can be streamed to two endpoints (private beta)**: This update allows you as an Enterprise owner to easily employ your choice of tools for log storage and analysis. Learn more about this beta and how to sign up in [our changelog](https://github.blog/changelog/2024-09-03-enterprise-audit-logs-can-be-streamed-to-two-endpoints-private-beta/).
* **Add repository permissions to custom organization roles**: You can now add repository permissions to custom organization roles, granting a specific level of access to all the repositories in your organization. [View the changelog for more information](https://github.blog/changelog/2024-08-29-add-repository-permissions-to-custom-organization-roles/). 

### 📚 Rousing Reads and Beyond: GitHub-produced goodies 🍭
* **Streamlining your MLOps pipeline with GitHub Actions and Arm64 runners**: @scottarbeit explores how Arm’s optimized performance and cost-efficient architecture, coupled with PyTorch, can enhance machine learning operations, from model training to deployment and learn how to leverage CI/CD for machine learning workflows, while reducing time, cost, and errors in the process in [this blogpost](https://github.blog/enterprise-software/ci-cd/streamlining-your-mlops-pipeline-with-github-actions-and-arm64-runners/).
* 📅 **Introduction to Copilot Webinar**: Are you curious to learn more about GitHub Copilot? Join us at this interactive introduction and find out why over 55,000 businesses worldwide are already taking advantage of the game-changing solution. [Sign up for a webinar that works for your schedule on our events page](https://resources.github.com/events/?type=webinar). 

Have you gotten GitHub Certified yet? If so, which program(s)?

**Comment by ghost** [comment]
> Have you gotten GitHub Certified yet? If so, which program(s)?

I managed to earn three GitHub Certifications this summer: GitHub Foundations, GitHub Administration, and GitHub Advanced Security. My experience with GitHub certification was phenomenal, and I thoroughly enjoyed the process of delving deep into GitHub Docs and having my knowledge validated through the GitHub Examination. I'm looking forward to my next certification journey, GitHub Co-Pilot, although I haven't booked it yet. Eventually, I also plan to pursue GitHub Actions certification eventually in the future.

**Comment by mapelvader** [comment]
qow gud jo

**Comment by apps4everyone** [comment]
News for Oktober?

**Reply by queenofcorgis** [reply]
@apps4everyone posting tomorrow - I'll comment here with a link 😄

**Comment by apps4everyone** [comment]
maybe also November, as October is almost finished 😄

**Comment by queenofcorgis** [comment]
Get our October Updates in the latest Enterprise Roundup posted in [this Discussion](https://github.com/orgs/community/discussions/142394).
